src/pages/user/Settings.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+
+import { Container, Row, Col, Form, InputGroup, Button } from 'react-bootstrap';
+
+export default class UserSettings extends Component {
+  constructor(...args) {
+    super(...args);
+
+    this.state = {
+      username: 'xXTeddyBearSlayer69Xx',
```

Comment:
🤔

Reply:
Sometimes you just need placeholder text and type the literal first thing that comes to your mind.

_This is one of those times._

Reply:
That's going to be my new username, once I drop mixnjuice

Reply:
I can't begin to describe how honored I feel to hear that